spec/codeowners/cli/config_spec.rb

```diff
@@ -12,29 +12,91 @@
   let(:config) { { config: git_config } }
 
   describe '#list' do
-    it 'fetch the owner from git config and show it' do
-      expect(cli).not_to receive(:help)
-      expect(git_config).to receive(:default_owner).and_return('@owner1').twice
-      expect do
+    before do
+      allow(git_config).to receive(:default_owner).and_return(default_owner)
+      allow(git_config).to receive(:default_organization).and_return(default_organization)
+    end
+
+    let(:default_owner) { '@owner1' }
+    let(:default_organization) { 'toptal' }
+
+    context 'with owner and organization' do
+      it 'fetches the owner from git config and shows it' do
+        expect { cli.list }.to output(/default_owner: "@owner1"\n/).to_stdout
+      end
+
+      it 'fetches the organization from git config and shows it' do
+        expect { cli.list }.to output(/default_organization: "toptal"\n/).to_stdout
+      end
+
+      it 'does not print help' do
+        expect(cli).not_to receive(:help)
         cli.list
-      end.to output("default_owner: \"@owner1\"\n").to_stdout
+      end
     end
 
-    it 'asks to provide a proper owner name' do
-      expect(cli).to receive(:help)
-      expect(git_config).to receive(:default_owner).and_return('').twice
-      expect do
+    context 'without owner name' do
+      let(:default_owner) { '' }
+
+      it 'prints help' do
+        expect(cli).to receive(:help)
         cli.list
-      end.to output("default_owner: \"\"\n").to_stdout
+      end
+
+      it 'shows empty owner' do
+        expect { cli.list }.to output(/default_owner: ""\n/).to_stdout
+      end
+    end
+
+    context 'without organization name' do
+      let(:default_organization) { '' }
+
+      it 'prints help' do
+        expect(cli).to receive(:help)
+        cli.list
+      end
+
+      it 'shows empty organization' do
+        expect { cli.list }.to output(/default_organization: ""\n/).to_stdout
+      end
     end
   end
 
   describe '#owner' do
-    it 'config the owner in the git config file' do
-      expect(git_config).to receive(:default_owner=).with('@owner1')
+    let(:new_owner) { '@owner1' }
+
+    before do
+      allow(git_config).to receive(:default_owner=)
+    end
+
+    it 'changes the owner in the git config file' do
+      expect(git_config).to receive(:default_owner=).with(new_owner)
+      cli.owner(new_owner)
+    end
+
+    it 'outputs success message' do
       expect do
-        cli.owner('@owner1')
+        cli.owner(new_owner)
       end.to output("Default owner configured to @owner1\n").to_stdout
     end
   end
+
+  describe '#organization' do
+    let(:new_org) { '@new-org' }
+
+    before do
+      allow(git_config).to receive(:default_organization=)
```

Comment:
Do we need this `before`? as we have it on line 92 also validating the `new_org`?

Reply:
In this scenario it does, because the test on line 96 also requires the default_organization method to be stubbed. I could've made a single test with both expectations to remove it, but I prefer to have a single expectation per spec

Reply:
Can you move it to the scenario that it makes sense? because for line 92 it still does not make sense.

Reply:
Sure! Moved and squashed fixups